src/hip_memory.cpp

```diff
@@ -2164,8 +2181,12 @@ hipError_t hipIpcCloseMemHandle(void* devPtr) {
         hipStatus = hipErrorInvalidValue;
     } else {
 #if USE_IPC
-        hsa_status_t hsa_status = hsa_amd_ipc_memory_detach(devPtr);
-        if (hsa_status != HSA_STATUS_SUCCESS) return hipErrorInvalidResourceHandle;
+        am_status_t am_status = hc::am_memtracker_remove(devPtr);
```

Comment:
Should this also be protected by LockedAccessor_CtxCrit as the attach/am_memtracker_add are - see line 2147 above?

Reply:
No